README.md

```diff
@@ -100,7 +100,7 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#BOHB">BOHB</a></li>
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#GPTuner">GP Tuner</a></li>
           </ul>
-          <li><b>Tuner for <a href="docs/en_US/CommunitySharings/NasComparision.md">NAS</a></b></li>
+          <li><b>Tuner for <a href="docs/en_US/AdvancedFeature/GeneralNasInterfaces.md">NAS</a></b></li>
```

Comment:
I don't think there needs a link here.

Reply:
But some users may be confused about the meaning of “NAS”. there is a simple description in this link

Reply:
the "general NAS interface" is not equal to "NAS"